appgate/__main__.py

```diff
@@ -118,13 +118,19 @@ def get_context(args: OperatorArguments,
     api_spec = generate_api_spec(spec_directory=Path(spec_directory) if spec_directory else None,
                                  secrets_key=secrets_key,
                                  k8s_get_secret=k8s_get_secret)
+
+    # When running in partial updates mode (target_tags defined) run also in cleanup mode
+    if target_tags and not cleanup_mode:
+        log.warning('[get_context] Activating cleanup mode since the operator is doing partial updates.')
+        cleanup_mode = True
```

Comment:
Why does partial updates imply cleanup mode?

Reply:
Well ... to avoid the case where you have entities in the system that depend on entities in the excluded tag but ... you the target tags dont manage them. 

Fixed to support both modes